tests/test_enterprise/api/test_views.py

```diff
@@ -381,6 +448,7 @@ def test_post_enterprise_customer_user_logged_out(self):
             FAKE_UUIDS[0],
             1,
             ENTERPRISE_CUSTOMER_COURSES_ENDPOINT,
+            False,
             True,
```

Comment:
@UmanShahzad The updates to the data for this test look great. I think we should add one more case: `is_staff`: `True`, `is_linked_to_enterprise`: `False`, `mocked_catalog_courses`: `fake_catalog_api.FAKE_CATALOG_COURSE_PAGINATED_RESPONSE`.

Reply:
:+1: